sailfishusermanager.cpp

```diff
@@ -777,3 +789,65 @@ void SailfishUserManager::removeFromGroups(uint uid, const QStringList &groups)
         }
     }
 }
+
+void SailfishUserManager::enableGuestUser(bool enable)
```

Comment:
This is now duplicating code from addUser quite a bit. Would be feasible to combine those common parts?